src/strategies/DSL/actions/marketSwapAction.ts

```diff
@@ -1,36 +1,50 @@
-import { MarketSwapActionArgs, PrimitiveArgs } from '@brinkninja/types'
+import { bigintToFeeAmount, toBigint, toTokenArgs } from '@brink-sdk/internal'
+import { UniV3Twap } from '@brink-sdk/oracles'
+import Token from '@brink-sdk/strategies/Token'
+import { MarketSwapActionArgs, PrimitiveArgs, TokenJSON } from '@brinkninja/types'
+
+const DEFAULT_TIME_INTERVAL = BigInt(1000)
```

Comment:
before I forget, can we change the default to `60` here and for the `price` condition?

Reply:
ok